emulators/qemu/files/patch-meson-disable-compiler-check.diff

```diff
@@ -0,0 +1,47 @@
+--- ./meson.build.orig	2024-12-12 13:32:42
++++ ./meson.build	2024-12-12 13:32:57
+@@ -309,25 +309,25 @@
+ # Compiler flags #
+ ##################
+ 
+-foreach lang : all_languages
+-  compiler = meson.get_compiler(lang)
+-  if compiler.get_id() == 'gcc' and compiler.version().version_compare('>=7.4')
+-    # ok
+-  elif compiler.get_id() == 'clang' and compiler.compiles('''
+-      #ifdef __apple_build_version__
+-      # if __clang_major__ < 15 || (__clang_major__ == 15 && __clang_minor__ < 0)
+-      #  error You need at least XCode Clang v15.0 to compile QEMU
+-      # endif
+-      #else
+-      # if __clang_major__ < 10 || (__clang_major__ == 10 && __clang_minor__ < 0)
+-      #  error You need at least Clang v10.0 to compile QEMU
+-      # endif
+-      #endif''')
+-    # ok
+-  else
+-    error('You either need GCC v7.4 or Clang v10.0 (or XCode Clang v15.0) to compile QEMU')
+-  endif
+-endforeach
++#foreach lang : all_languages
```

Comment:
Just remove the lines, do not comment them out. This makes it more complex to see what the patch is doing.